### ISSUE TITLE
Import fix creates incorrect URL for symbols from preact/hooks

### ISSUE THREAD

**Issue by cmlenz** [microsoft/TypeScript#37542]
**TypeScript Version:**  3.8.3

**Search Terms:**
import preact/hooks preact/hooks/src

**Code**

```tsx
import {h} from 'preact';

function App(): void {
  const value = useMemo(() => 'Hello');
  return <div>{state}</div>;
}
```

Run missing import fix on `useMemo`.

**Expected behavior:**

The quick fix should add the import line

```ts
import {useMemo} from 'preact/hooks';
```

**Actual behavior:**

The quick fix actually adds the incorrect import line

```ts
import {useMemo} from 'preact/hooks/src';
```

This will cause the source files from Preact to be bundled instead of the transpiled files, which will cause various kinds of breakage (see https://github.com/preactjs/preact/issues/1757 or https://github.com/preactjs/preact/issues/1373).

**Comment by sheetalkamat** [comment]
Please demonstrate the problem with detail since just creating the file does not give code fix as program doesn't have anything except `jsx.d.ts` and `index.d.ts` from `preact` included. Also try out typescript@next to see if the issue repros.

```
c:/Typescript/built/local/lib.d.ts
	c:/Typescript/built/local/lib.es5.d.ts
	c:/Typescript/built/local/lib.dom.d.ts
	c:/Typescript/built/local/lib.webworker.importscripts.d.ts
	c:/Typescript/built/local/lib.scripthost.d.ts
	c:/temp/temp/node_modules/preact/src/jsx.d.ts
	c:/temp/temp/node_modules/preact/src/index.d.ts
	c:/temp/temp/a.tsx
```

**Comment by cmlenz** [comment]
I have prepared a minimal reproduction for the issue we are seeing here:  

https://github.com/mediatis/ts37542-preact-autoimport

It turns out that I needed to have a second file in the project that imports from `preact/hooks`. When that is in place, attempting to quick fix a missing import in another file will suggest `preact/hooks/src`.

The repo above contains step by step instructions for reproduction in the README. Also it is set up to use typescript@next.
